src/main/scala/io/iohk/ethereum/consensus/ethash/validators/OmmersValidator.scala

```diff
@@ -36,9 +37,10 @@ object OmmersValidator {
 
   object OmmersError {
     case object OmmersLengthError extends OmmersError
-    case object OmmersNotValidError extends OmmersError
+    case class OmmersHeaderError(error: Option[BlockHeaderError]) extends OmmersError
```

Comment:
This will probably have to contain a list of header errors.

see https://github.com/input-output-hk/mantis/pull/948/files#diff-4ffb3286be2984d1679ee76bbb2e5e404f64147f1d77ab0eb95f8031a1379635R81